rs/replicated_state/src/canister_state/system_state.rs

```diff
@@ -292,9 +299,25 @@ impl TaskQueue {
         on_low_wasm_memory_hook_status: OnLowWasmMemoryHookStatus,
         canister_id: &CanisterId,
     ) -> Self {
+        let mut mut_queue = queue;
+
+        // Extraction of paused_or_aborted_task from queue will be removed in the follow-up EXC-1752 when
```

Comment:
Is the plan to do EXC-1752 immediately afterwards? And I assume the plan there is to introduce a proto field field for `paused_and_aborted_task`?

Non-trivial serialize/deserialize code is always playing with fire. If you get it wrong for some poorly tested edge-case the replica will crash. 

I'm ok with it if the  follow-up is planned immediately and we are sure about my other question about the front of queue.